ash/src/extensions/ext/debug_marker.rs

```diff
@@ -6,19 +6,17 @@ use std::mem;
 
 #[derive(Clone)]
 pub struct DebugMarker {
-    debug_marker_fn: vk::ExtDebugMarkerFn,
+    handle: vk::Device,
```

Comment:
The general consensus was is that we only hold on to the instance/device `handle` if it is actively used in the functions.

You'll have to change the function signatures of ie. `debug_marker_set_object_name` to remove `device: vk::Device,` from it since this is now available as `self.handle`.

More importantly, it prevents callers from using these functions with a different device than it was originally retrieved with through `get_device_proc_addr` and simplifies the calling some, too.

Reply:
Done